android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -353,7 +370,9 @@ public void run() {
 
                     DefaultAllocator allocator = new DefaultAllocator(true, C.DEFAULT_BUFFER_SEGMENT_SIZE);
                     DefaultLoadControl defaultLoadControl = new DefaultLoadControl(allocator, minBufferMs, maxBufferMs, bufferForPlaybackMs, bufferForPlaybackAfterRebufferMs, -1, true);
-                    player = ExoPlayerFactory.newSimpleInstance(getContext(), trackSelector, defaultLoadControl);
+                    DrmSessionManager<FrameworkMediaCrypto> drmSessionManager = drmSessionManager = buildDrmSessionManager(self.drmUUID, self.srcUri, self.drmLicenseUrl, self.drmToken);
```

Comment:
Duplicated `drmSessionManager =`?
```suggestion
                    DrmSessionManager<FrameworkMediaCrypto> drmSessionManager = buildDrmSessionManager(self.drmUUID, self.srcUri, self.drmLicenseUrl, self.drmToken);
```